homeassistant/const.py

```diff
@@ -181,9 +181,10 @@
 DEVICE_CLASS_BATTERY = 'battery'
 DEVICE_CLASS_HUMIDITY = 'humidity'
 DEVICE_CLASS_ILLUMINANCE = 'illuminance'
+DEVICE_CLASS_POWER = 'power'
```

Comment:
Please revert this change.

If you want a device class, you need open a issue on architecture repository.

Reply:
Hi, i reverted the change.